CC/Sounder/include/Radio.h

```diff
@@ -1,25 +1,23 @@
-/*
- Copyright (c) 2018-2022, Rice University 
- RENEW OPEN SOURCE LICENSE: http://renew-wireless.org/license
- 
+/** @file Radio.h
+  * @brief Declaration file for the Radio class.
+  * 
+  * Copyright (c) 2018-2022, Rice University 
+  * RENEW OPEN SOURCE LICENSE: http://renew-wireless.org/license
 */
+#ifndef RADIO_H_
+#define RADIO_H_
 
-#include <SoapySDR/Device.hpp>
-#include <SoapySDR/Time.hpp>
+#include <cstdlib>
+#include <vector>
 
+#include "SoapySDR/Device.hpp"
```

Comment:
Do you really need to include all these soapy headers?

Reply:
these are still necessary since it relies on some of the flags created by those soapy headers.